src/modules/personenkontext/domain/personenkontexte-update.ts

```diff
@@ -79,6 +79,7 @@ export class PersonenkontexteUpdate {
         );
     }
 
+    /* eslint-disable no-await-in-loop */
```

Comment:
the getSentPersonenkontexte() for,,,loop needs to be refactored as we talked, is there a ticket for that?

Reply:
yes there is a ticket :) SPSH-1082